### DISCUSSION TITLE
오픈채팅방 해외유저 구분하는 방법

### DISCUSSION THREAD

**Discussion by ZZLOCO** [storycraft/node-kakao#645] [Ideas]
오픈채팅방 해외유저 구분하는 방법

1. 오픈채팅방의 실제 프로필을 구해 유저 정보를 받아오며 국가를 확인하는 방법
2. 오픈채팅방에서 선물하기를 이용하여 일반 유저와 해외유저를 구분하는 방법 (국가 확인X)

![123](https://user-images.githubusercontent.com/89644716/132813940-c55bb107-0559-4ddf-b38f-fd2155eda7ce.png)

**Comment by ha1fstack** [comment]
실제 프로필은 예전에 로직 변경점이 있었던걸로 아는데 바뀐건 어떤식인가요?

**Reply by gd1422333** [reply]
2020년 8월 이후부터 변경된걸로 알고있습니다.

[a comment by an automated account is not shown]
